source/includes/harvest/_job_posts.md

```diff
@@ -250,3 +252,96 @@ questions | If present, will return an array of questions associated with this j
 
 <br>
 [See noteworthy response attributes.](#the-job-post-object)
+
+## List multiple job posts for a job
```

Comment:
Can we drop the word 'multiple' to keep the title short for the navigation bar on the left?  `

```
## List job posts for a job
```